public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -222,7 +222,7 @@ export function GeneralSettingsUnconnected({
           </Field>
         </CollapsableSection>
 
-        <Box marginTop={3}>{dashboard.meta.canDelete && <DeleteDashboardButton dashboard={dashboard} />}</Box>
+        <Box marginTop={3}>{dashboard.meta.canDelete && <DeleteDashboardButton />}</Box>
```

Comment:
Let the component use `getCurrent` to use the overriden `getDashboardSrv().getCurrent()` from scenes. The API is weird without any props tbh.